plugin/sampling/strategystore/static/strategy_store.go

```diff
@@ -126,9 +171,27 @@ func (h *strategyStore) updateSamplingStrategy(bytes []byte) error {
 
 // TODO good candidate for a global util function
 func loadStrategies(strategiesFile string) (*strategies, error) {
-	if strategiesFile == "" {
-		return nil, nil
+	// Check if it's a URL.
+	if ok := isURL(strategiesFile); ok {
```

Comment:
there is an existing redundancy in the original source, please consolidate it. You should not need to check isURL twice in this source file.